Added check for empty units

h-mayorquin opened the pull request:
Short PR to add a check for empty units in the behavioral interface. This is because the spatial series in `pynwb` expects string by default and produces an assertion error with the None type.

CodyCBakerPhD commented:
@h-mayorquin can you include the re-nesting of orientation here too?

CodyCBakerPhD commented:
And also append "_track" to the position series name; i.e., `linear` -> `linear_track`

h-mayorquin commented:
I added track to the analysis and nested the orientation also added a different description more in line with the methods of the paper.

What I could not find and I am out of ideas is what description add to the Error.

CodyCBakerPhD commented:
@h-mayorquin Could always send one last email to Tingley himself to see what he would describe that field as. Thanks for staying up late to get this finished!

CodyCBakerPhD commented:
Per Tingley's response: `description="Estimated error for marker tracking from optitrack system."`

CodyCBakerPhD commented:
@h-mayorquin LGTM; merging when tests are done, then we should be ready for final conversion. Were you still getting errors with `parfor`?
